src/tests/Equinor.ProCoSys.IPO.Command.Tests/InvitationCommands/EditInvitation/EditInvitationCommandHandlerTests.cs

```diff
@@ -49,7 +49,8 @@ public class EditInvitationCommandHandlerTests
         private const string _newDescription = "Test description 2";
         private const string _firstName = "Ola";
         private const string _lastName = "Nordmann";
-        private const DisciplineType _type = DisciplineType.DP;
+        private const DisciplineType _typeDp = DisciplineType.DP;
+        private const DisciplineType _typeMdp = DisciplineType.MDP;
         private readonly Guid _meetingId = new Guid("11111111-2222-2222-2222-333333333333");
         private Invitation _invitation;
```

Comment:
Rename this to _dpInvitation, make a _mdpInvitatinon to
Create tests to show that we get exception if me "convert" a dp to mdp (and vise verca) to have illegal scope (as discussed)